rust/headless-client/src/lib.rs

```diff
@@ -49,8 +42,10 @@ pub mod platform;
 #[path = "windows.rs"]
 pub mod platform;
 
+pub use ipc_service::{ipc, run_only_ipc_service, ClientMsg as IpcClientMsg};
```

Comment:
Just an inspirational comment, no need for action right now:

I've found renaming imports to be a bit tedious to work with. Jumping to definition from the usage site suddenly lands you at a differently named type which can be confusing. Instead, I usually try to structure modules such that the public API of the module - together with its name - provides an unambiguous way of referring to the types. So in this case for example, we are importing:

- `ipc`
- `run_only_ipc_service`
- `ClientMsg`

Some ideas on what we could do instead:

- Rename `ipc_service` to just `ipc` and then refer to `ClientMsg` as `ipc::ClientMsg`
- Rename `run_only_ipc_service` to `run_service` and then you can refer to it as `ipc::run_service`
- Rename the _inner_ `ipc` module to something like `protocol` maybe